system/ee/ExpressionEngine/View/_shared/add-on-card.php

```diff
@@ -10,6 +10,18 @@
 		<?php if (!empty($addon['description'])): ?>
 		<p class="add-on-card__desc" title="<?= $addon['description'] ?>"><?= $addon['description'] ?></p>
 		<?php endif; ?>
+
+		<?php if ($addon['installed'] && $addon['developer'] == 'EEHarbor') : ?>
```

Comment:
Is this something we intended to include? We tried to avoid previously direct mentions of `EEHarbor` in the code, because even though it's almost same team of developers, technically this is different vendor, and we try to not include vendor-specific stuff